pkg/oauth/apis/oauth/validation/validation_test.go

```diff
@@ -746,3 +746,38 @@ func TestValidateAuthorizeTokensUpdate(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateUserNameField(t *testing.T) {
+	fieldArg := field.NewPath("metadata", "name")
+	valid := "kube:admin"
```

Comment:
nit: `validUsername`. Be verbose with variable names

Reply:
Done